test/e2e/node/crictl.go

```diff
@@ -42,24 +45,32 @@ var _ = SIGDescribe("crictl", func() {
 		nodes, err := e2enode.GetBoundedReadySchedulableNodes(ctx, f.ClientSet, maxNodes)
 		framework.ExpectNoError(err)
 
-		testCases := []string{
-			"crictl version",
-			"crictl info",
```

Comment:
Why `crictl info` testing was dropped?

Reply:
Before the change the command output wasn't tested,only that stdout or stderr had contents.

`critctl version` outputs info about the container runtime:

```
Version:  0.1.0
RuntimeName:  containerd
RuntimeVersion:  1.7.27
RuntimeApiVersion:  v1
```

So I thought that checking at least that critctl connected to containerd and got its version was good enough.

I can also check properties of the JSON outputted by `crictl info`, open to more suggestions.